cmd/util/cmd/execution-state-extract/checkpoint_status.json

```diff
@@ -0,0 +1,3 @@
+{
+ "succeeded": true
+}
```

Comment:
is this something added by this PR or accidentally here.

Reply:
Oh that must have been generated by the tests.